Sofa/framework/Helper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -186,39 +167,6 @@ const std::map<std::string, ComponentChange, std::less<> > uncreatableComponents
     {"TriangleModel", Removed("v19.12", "v20.06")},
     {"TTriangleModel", Removed("v19.12", "v20.06")},
 
-    /***********************/
-    // REMOVED SINCE v18.12
-
-    // SofaBoundaryCondition
-    {"BuoyantForceField", Removed("v17.12", "v18.12")},
```

Comment:
I'm not in favor of removing things that are cheap and helps users

Reply:
I think removing content because the *compiler/linker* does not like it is a bad answer to a legitimate problem. 

The "target being tool large problem" have its root in the fact that simplicity we have decided to hard-code in the code/binary file the "data".  This is a very questionable practice that was tested for simplicity at time of introduction of ComponentChange but the proper design is to always have code that is "executable" and "data" that are stored in a separated file (a data file). 

So the the proper answer for the "target starts being too large" is to move al the data in a data file and load it to initialize the ComponentChange. Making a json loader for such thing is super easy and should be done.

Reply:
An issue has been open to provide more discussion on that topic.